### ISSUE TITLE
Help!!!!!!!!!!   tf.keras.layers.Conv2d and slim.conv2d have different result(Though slim.conv2d had been deprecated)

### ISSUE THREAD

**Issue by zihao-lu** [tensorflow/tensorflow#41649]
TF 1.15.0
Ubuntu 16.0
python 3.7.6
with same parameter(kernel , bias, padding etc.) tf.keras.layers.Conv2d and slim.conv2d have different output. Though the different is little, about 0.000001,  when i copy the model parameters training with slim.conv2d to the model builded with tf.keras.layers.Conv2d, I got a completely output

**Comment by amahendrakar** [comment]
@zihao-lu,
In order to expedite the trouble-shooting process, could you please provide the complete code to reproduce the issue reported here and also the dataset you are using. Thanks!
